### DISCUSSION TITLE
Cannot reverify my Academic Affiliation

### DISCUSSION THREAD

**Discussion by YusufOzmen01** [community/community#56129] [GitHub Education]
### Hi everyone,

I'm a Turkish high school student. Two years ago I applied to Github Student Pack using my student certificate and it got approved. Now I'm trying to reapply using my student certificate again (though I got it like 2 minutes before submitting an application) and it got rejected. 7 times. Tried my student id aswell and didn't work. I literally don't know why this is happening. I'd appreciate any help

**Comment by Rezvanpm** [comment]
Unfortunately I have similar problem but GitHub support isn’t any response to my ticket.

**Comment by mrepol742** [comment]
Did you used your school email address? Github Edu is now updated and they are requiring you to use school email address.

For example:
If the school address is https://example.com you should use yourname@example.com or yourschoolid@example it depends on your school.

**Comment by samus-aran** [comment]
Hi @YusufOzmen01 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
